bower.json

```diff
@@ -13,7 +13,7 @@
     "chartist": "^0.9.8",
     "font-awesome": "4.7.0",
     "html5shiv": "latest",
-    "jquery": "1.x",
+    "jquery": "3.x",
```

Comment:
Do we need this if we are committing jQuery?

Reply:
First steps, keeping the change footprint small